lib/emerald/nodes/list.rb

```diff
@@ -4,22 +4,15 @@
 require 'treetop'
 require_relative 'node'
 
-# TODO: Refactor this and update doc comment
-# Multiple head elements I think
-class List < Node
+# Special rule for lists of images, styles, and scripts
+class ListSpecial < Node
```

Comment:
Instead of `ListSpecial` and `List`, can we rename to something like `AttrList` and `ValueList` to indicate that one has attribute hashes, and the other transforms a list of individual values?